drivers/SmartThings/zigbee-thermostat/src/init.lua

```diff
@@ -164,6 +164,63 @@ local thermostat_mode_setter = function(mode_name)
   end
 end
 
+local thermostat_setpoint_min_cool = nil
+local thermostat_setpoint_max_cool = nil
+local thermostat_setpoint_min_heat = nil
+local thermostat_setpoint_max_heat = nil
```

Comment:
Unfortunately, local variables in here will be in scope for _all_ devices using this driver. In order to use device-specific values you'll need to use the data store with `device:set_field()`

Reply:
I pushed an update to use `device:set_field` and `device:get_field` instead of the local variables.